api/valor_api/backend/core/llm_instructions_analysis.py

```diff
@@ -0,0 +1,573 @@
+def _generate_claims_instruction(text: str) -> str:
```

Comment:
all of the code in this file is not being used since you decided to go with the non-analysis version, right?

nit: it might make sense to save this code locally and delete it from the repo